workload/master_config.yaml

```diff
@@ -0,0 +1,43 @@
+s3api:
+  # TEST_ID: TEST-35748
+  object_size:
+    start: 0Kib
+    end: 100Kib
+  part_range: #Multipart Specific
+    start: 50
+    end: 100
+  min_runtime_to_pass: 2h
+  sessions_per_node: 1
+  range_read: 100bytes
+  tool: s3api
+  operation: bucket/object_range_read/object_fix_size/object_random_size/copy_object/multipart/multipart_partcopy
+  wait_on_operation: True/False # True: Wait till operation completes to mark pass,
+  #if False , test will be marked pass as per minimum run time
+
+mix_io:
+  # TEST_ID: TEST-40042
+  object_size: #(Single input, List, and Dictionary{key:value})  object size =
+  # for 10000 samples(user configurable) and distribution input may be like {1kb :10%, 100kb: 20%, 1000kb: 30%, 10000kb: 40%}
+    start: 0bytes
+    end: 1Gib
+  write_percentage: 100  # Percentage of data to fill the storage.
+  read_percentage: 100  # Percentage of data to be used to read from storage.
+  delete_percentage: 0  # Percentage of data to be deleted from storage.
+  cleanup_percentage: 0  # Percentage of data once reached then cleanup all data.
+  #  If None then it will be fetched from cluster(cortx specific.)
+  total_storage_size: None  # Total available storage from cluster or user to be used.
+  min_runtime_to_pass: 30d  # Minimum execution durations to mark workload pass.
+  sessions_per_node: 20  # Number of sessions per node.
```

Comment:
Can we keep sessions_per_node 1 by default.

Reply:
Sure, Updated accordingly.